buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/BuildPlugin.groovy

```diff
@@ -84,6 +88,32 @@ class BuildPlugin implements Plugin<Project>  {
     }
 
     private static void configureConfigurations(Project project) {
+        if (project != project.rootProject) {
+            // Set up avenues for sharing source files between projects in order to create embedded Javadocs
+            // Import source configuration
+            Configuration sources = project.configurations.create("sources")
```

Comment:
would `additionalSources` read better ?

Reply:
Yeah I think you're on to something there. I'll make that change.